src/suit/html/slate.html

```diff
@@ -4,18 +4,19 @@
 <head>
     <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
     <meta name="apple-mobile-web-app-capable" content="yes" />
-    <title>LSST Portal</title>
+    <title>Rubin Science Platform Portal</title>
 
     <script>
         var pOpts = window.firefly && window.firefly.options || {};
         window.firefly = {
             app: {
                 template: 'FireflySlate',
                 menu: [
-                    {label:'Images', action:'ImageSelectDropDownSlateCmd'},
-                    {label:'Catalogs', action:'IrsaCatalogDropDown'},
-                    {label:'Charts', action:'ChartSelectDropDownCmd'},
-                    {label:'Upload', action: 'FileUploadDropDownCmd'},
+                    {label: 'RSP TAP Search', action: 'TAPSearch'},
+                    {label: 'External Images', action: 'ImageSelectDropDownCmd'},
+                    {label: 'External Catalogs', action: 'MultiTableSearchCmd'},
```

Comment:
This looks right.  But, not sure if `MultiTableSearchCmd` includes TAP by default.  You may need to verify.